third_party/silabs/SiWx917_sdk.gni

```diff
@@ -68,16 +68,22 @@ template("siwx917_sdk") {
 
       # siwx917_soc component
       "${wifi_sdk_root}/components/siwx917_soc/inc",
+      "${wifi_sdk_root}/components/siwx917_soc/boards/config/brd4325b",
```

Comment:
This looks to be incorrect brd4325b for all boards?

Reply:
that works actually. but yeah, will update the board specific path